core/src/components/select/select.tsx

```diff
@@ -41,6 +42,7 @@ import type { SelectCompareFn } from './select-interface';
 @Component({
   tag: 'ion-select',
   styleUrls: {
+    base: 'select.scss',
```

Comment:
When opening a select of any interface with `use-base`, I get a couple warnings in the console:
![warnings](https://i.gyazo.com/5f9f34060401411e1bf0dad0a864be66.png)

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/26669/commits/3d050538a2a3fb8c9d798aa91126a06e0348e560